docs/source/platform_technical_documentation/ade_custom_extensions/maap_libs.ipynb

```diff
@@ -0,0 +1,58 @@
+{
```

Comment:
<p>Recommend:</p><ul><li>Capitalizing instances of "jupyter"</li><li>initailizes -&gt; initializes</li></ul>
<br/>

 _Reply via <a href='https://app.reviewnb.com/MAAP-Project/maap-documentation/pull/146/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>